### PR TITLE
Add shout chat

### LINKED ISSUE #144: Implement shout
Implement the shout system. Shout == whole map.

### PR DESCRIPTION
**Pull request by rminderhoud**
Add shout chat. Closes #144 and progresses #40

---
This change is [Reviewable](https://reviewable.io/reviews/dev-osrose/osirose-new/169)

### CODE REVIEW

File: src/map/src/chat/shout_chat.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#include "cli_shout_chat.h"
+#include "srv_shout_chat.h"
+
+using namespace RoseCommon;
+using namespace RoseCommon::Packet;
+
+void Chat::shout_chat(EntitySystem& entity_system, Entity entity, const CliShoutChat& packet) {
+    auto logger = Core::CLog::GetLogger(Core::log_type::GENERAL).lock();
+    logger->trace("Chat::shout_chat()");
+    logger->trace("entity {}, message {}", entity, packet.get_message());
```
Comment (L3nn0x) [inline src/map/src/chat/shout_chat.cpp]: I would have used debug here instead of trace.

**Review by L3nn0x — DISMISSED** [review]

File: src/map/src/chat/shout_chat.cpp (hunk 1 above)
Comment (RavenX8) [reply src/map/src/chat/shout_chat.cpp]: line 15 only, line 14 would be a trace

Comment (L3nn0x) [reply src/map/src/chat/shout_chat.cpp]: Yeah, but I checked the normal chat and we put the message as a trace instead of debug. So it could be either

Comment (rminderhoud) [reply src/map/src/chat/shout_chat.cpp]: Updated to a debug message and rebased

**Review by L3nn0x — APPROVED** [review]

**Pull request merged**